manager/integration/tests/test_ha.py

```diff
@@ -2142,8 +2142,96 @@ def test_reuse_failed_replica():
     11. Verify the failed replica (in step 5) will be reused.
     12. Verify the volume r/w still works fine.
     """
-    pass
+    long_wait = 60*60
+    short_wait = 3
+
+    replenish_wait_setting = \
+        client.by_id_setting(SETTING_REPLICA_REPLENISHMENT_WAIT_INTERVAL)
+    client.update(replenish_wait_setting, value=str(long_wait))
+
+    replica_node_soft_anti_affinity_setting = \
+        client.by_id_setting(SETTING_REPLICA_NODE_SOFT_ANTI_AFFINITY)
+    client.update(replica_node_soft_anti_affinity_setting, value="false")
+
+    vol = create_and_check_volume(client, volume_name)
+    host_id = get_self_host_id()
+    vol = vol.attach(hostId=host_id)
+    vol = common.wait_for_volume_healthy(client, volume_name)
+    data = {
+        'pos': 0,
+        'content': common.generate_random_data(16*Ki),
+    }
+    common.write_volume_data(vol, data)
+
+    current_host = client.by_id_node(id=host_id)
+    client.update(current_host, allowScheduling=False)
+
+    vol = client.by_id_volume(volume_name)
+    assert len(vol.replicas) == 3
+    other_replicas = []
+    for r in vol.replicas:
+        if r["hostId"] == current_host.name:
+            replica_1 = r
+        else:
+            other_replicas.append(r)
+    replica_2, replica_3 = other_replicas
+
+    for filenames in os.listdir(replica_1.dataPath):
+        if filenames.endswith(".img"):
+            with open(os.path.join(replica_1.dataPath, filenames), 'w') as f:
+                f.write("Longhorn is the best!")
+
+    crash_replica_processes(client, core_api, volume_name,
+                            replicas=[replica_1],
+                            wait_to_fail=False)
+
+    for i in range(30):
```

Comment:
We only want to wait to 1 minute to verify that that there is no new replica. If we use constant it would be 2.5 minutes which is unnecessary.

I have added the comment. Thanks

Reply:
You can add a new constant in this python file.

BTW, the comments and the update for `test_reuse_failed_replica` are added to the commit `Implement test_reuse_failed_replica_with_scheduling_check test`

Reply:
Thank you. Fixed